ingestion_server/ingestion_server/ingest.py

```diff
@@ -326,26 +320,48 @@ def reload_upstream(
         downstream_cur.execute(copy_data)
 
     next_step = (
-        "starting data cleaning"
+        "_Next: {starting data cleaning}_"
         if table == "image"
-        else "re-applying indices & constraints"
+        else "Finished refreshing table"
     )
-    slack.verbose(f"`{table}`: Data copy complete | _Next: {next_step}_")
+    slack.verbose(f"`{table}`: Data copy complete | {next_step}")
 
     if table == "image":
         # Step 5: Clean the data
         log.info("Cleaning data...")
         clean_image_data(table)
         log.info("Cleaning completed!")
         slack.verbose(
-            f"`{table}`: Data cleaning complete | "
-            f"_Next: re-applying indices & constraints_"
+            f"`{table}`: Data cleaning complete | " f"Finished refreshing table"
         )
 
-    log.info("Starting post-cleaning work...")
+    downstream_db.close()
+    log.info(f"Finished refreshing table '{table}'.")
+    _update_progress(progress, 100.0)
+
+
+def promote_api_table(
+    table: str,
+    progress: multiprocessing.Value = None,
+):
+    """
+    This runs after ``refresh_api_table``. The process involves the following steps.
+
+    6. Recreate indices from the original table: ``_generate_indices``
+    7. Recreate constraints from the original table: ``_generate_constraints``
+    8. Promote the temp table and delete the original: ``get_go_live_query``
+
+    :param table: the upstream table to copy
+    :param progress: multiprocessing.Value float for sharing task progress
+    """
+
+    log.info(f"`{table}`: Starting table promotion | _Next: recreate-indices_")
+    downstream_db = database_connect()
+    _, index_mapping = _generate_indices(downstream_db, table)
```

Comment:
This is also defined on L365

Reply:
Good find! Removed the redundant line in a8468c7.